### ISSUE TITLE
Options with argument placeholders are wrongly rendered in the HTML manpage of rpmbuild(8)

### ISSUE THREAD

**Issue by brlin-tw** [rpm-software-management/rpm#3502]
**Describe the bug**

Refer to the following screenshot for the issue:

![Screenshot](https://github.com/user-attachments/assets/001ec5cc-1406-40f5-b273-a58a38690b62)

The literal asterisks shouldn't be there, I think some formatting is broken.

**To Reproduce**
1. Browse <https://rpm-software-management.github.io/rpm/man/rpmbuild.8.html#general-options>
2. Locate options with argument placeholders like `--rcfile`
3. Error encountered

**Expected behavior**

No literal asterisks and the option argument placeholders formatted in italics or other emphasize formatting.

**Environment**

Mozilla Firefox 133.0.3

**Comment by pmatilai** [comment]
Indeed, thanks for the report. Unsurprisingly, similar issues exist in the other manpages too, so we'd better go through them all.
